Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -319,6 +365,40 @@ def review(self, pull_request, comment=None, approve=None):
             if comment:
                 body['body'] = comment
 
+            if diff_comments:
+                # Assume that every comment made on the same line as an existing comment
+                # is intended as a reply to the existing comment
+                existing_comment_ids = self._diff_comments(pull_request, ids=True)
+                body['comments'] = []
+                for file, line_comments in diff_comments.items():
+                    for position, comments in line_comments.items():
+                        existing_ids = existing_comment_ids.get(file, {}).get(position, [])
+                        if existing_ids:
+                            is_successful &= self._make_comment(pull_request, dict(
+                                in_reply_to=existing_ids[-1],
```

Comment:
This is a good point, and this is actually worse in BitBucket (because you can reply to replies in BitBucket). I'm going to add a FIXME for now. Addressing this won't be simple, we have to carry around comment IDs with us, or we have to include some sort of "reply to" logic. The real issue is how our callers do this, so I'm inclined to postpone resolving this problem until we understand how our callers will work better. There is also the fact that, in practice, users very rarely comment on a PR line without replying to the existing comment, so this is an edge case we're unlikely to be bothered by in practice.